Monika After Story/game/script-farewells.rpy

```diff
@@ -40,6 +40,8 @@ init -1 python in mas_farewells:
     #(Used in the generic rtg label if we failed to generate a file)
     dockstat_failed_io_still_going_ask_label = None
 
+    MOST_USED_SORT_KEY = lambda x: x.shown_count
```

Comment:
this is useful for mainly event, so rather than a constant, put this on the Event class as a static function